app/assets/stylesheets/arclight/modules/layout.scss

```diff
@@ -4,6 +4,27 @@
   margin-bottom: $spacer;
 }
 
+.al-show-breadcrumb .breadcrumb {
+  background-color: transparent;
+  padding: 0;
+}
+
+.breadcrumb-item {
+  flex: 1 0 100%;
+
+  + .breadcrumb-item.al-bc-item {
+    &::before {
+      content: '»';
+    }
+  }
+}
+
+@for $i from 1 to 12 {
```

Comment:
Great catch. I'm updating this to be `14` which I think should be correct.

0 - home (not counted because of matching `+ .class` selector ... i think)
1 - collections
2 - document/collection
3 - 14 (1 - 12 levels of components?)